docker-compose.yml

```diff
@@ -144,6 +150,13 @@ services:
       bouldernet:
         ipv4_address: 10.77.77.17
 
+  bpkilint:
+    image: ghcr.io/digicert/pkilint
```

Comment:
This will pull whatever is `latest`, I think we should pin the version instead and make a plan to review updates.

Reply:
Good call. I've pinned this to [v0.10.1](https://github.com/digicert/pkilint/pkgs/container/pkilint/207002815?tag=v0.10.1), which is currently the latest numbered release.